docs/dsl_inspec.md

```diff
@@ -215,7 +215,7 @@ control 'windows-base-101' do
 end
 ```
 
-### Exclude specific test
+### Use `only_if` to exclude a specific test
```

Comment:
In this whole section, the term `test` should possibly be `control`

I would use the term `test` to refer to refer to individual `it` assertions... but maybe I'm wrong on that.